src/components/CurriculumComponents/CurriculumDownloads/CurriculumDownloads.test.tsx

```diff
@@ -147,48 +150,42 @@ describe("Component - Curriculum Header", () => {
     await userEvent.click(getByTestId("checkbox-download")!);
     await userEvent.click(getByTestId("termsCheckbox").querySelector("label")!);
     await userEvent.click(getByTestId("loadingButton"));
-    await waitFor(
-      () => {
-        expect(createAndClickHiddenDownloadLink).toHaveBeenCalledWith(
-          "https://test-url.com/download?type=curriculum-map&extension=pdf&id=test-subject",
-        );
-
-        expect(getByTestId("downloadSuccess")).toBeInTheDocument();
-      },
-      { interval: 100, timeout: 2000 },
+    expect(createAndClickHiddenDownloadLink).toHaveBeenCalledWith(
+      frenchResource.url,
     );
+    expect(getByTestId("downloadSuccess")).toBeInTheDocument();
   });
 
   test("generates error when Hubspot fails", async () => {
-    (useHubspotSubmit as jest.Mock).mockImplementation(() => ({
-      useHubspotSubmit: jest.fn().mockImplementation(() => {
-        return {
-          onHubspotSubmit: jest.fn().mockImplementation(() => {
-            throw new Error("Test Error");
-          }),
-        };
-      }),
-    }));
```

Comment:
This wasn't working as it was originally intended. Removed and changed the approach to where there is a variable that can be set to determine if the useHubspotSubmit mock fails or not. (lines 21 - 28)